server/run.py

```diff
@@ -6,5 +6,5 @@
 if __name__ == '__main__':
     import os  
     port = int(os.environ.get('PORT', 5000)) 
-    routes.app.run(host='0.0.0.0', port=port)
+    routes.app.run(host='127.0.0.1', port=port)
     
```

Comment:
add a newline here

Reply:
There's already a new line, line: 10, or does it need two new lines?